WebDriverAgentLib/Utilities/FBConfiguration.m

```diff
@@ -109,12 +108,15 @@ + (BOOL)shouldUseSingletonTestManager
   return FBShouldUseSingletonTestManager;
 }
 
-+ (void)setShouldLoadSnapshotWithAttributes:(BOOL)value {
-  FBShouldLoadSnapshotWithAttributes = value;
-}
-
 + (BOOL)shouldLoadSnapshotWithAttributes {
-  return FBShouldLoadSnapshotWithAttributes;
+  static BOOL shouldLoadSnapshotWithAttributes = NO;
```

Comment:
shouldn't these static vars be defined outside the method?

Reply:
It's only accessed in this method. I don't see why it should be defined outside of this method